src/config/config.go

```diff
@@ -2,7 +2,15 @@ package config
 
 type Config struct {
```

Comment:
@nicobarra please, we need a card with this implementation in jobs and synchronizers project

Reply:
Ok, done 👍